crates/apub/src/api/list_posts.rs

```diff
@@ -30,8 +30,11 @@ pub async fn list_posts(
   let page = data.page;
   let limit = data.limit;
   let community_id = if let Some(name) = &data.community_name {
-    Some(resolve_actor_identifier::<ApubCommunity, Community>(name, &context, &None, true).await?)
-      .map(|c| c.id)
+    Some(
+      resolve_actor_identifier::<ApubCommunity, Community>(name, &context, &local_user_view, true)
```

Comment:
This is the problem, without auth it would only lookup in the local db and not make a webfinger request. No idea why it was hardcoded to None.